webviz_config/plugins/_data_table.py

```diff
@@ -10,18 +10,18 @@
 
 
 class DataTable(WebvizPluginABC):
-    """### Data table
-
-Adds a table to the webviz instance, using tabular data from a provided csv file.
+    """Adds a table to the webviz instance, using tabular data from a provided csv file.
```

Comment:
Either have **table** in bold face here (and as a **"standard**" for a keyword in the description of all plugins), or skip the bold face for the **banner image** above. Think maybe the latter is the best.
```suggestion
    """Adds a **table** to the webviz instance, using tabular data from a provided csv file.
```